R/compiled.R

```diff
@@ -109,5 +144,42 @@ line_coverages <- function(source_file, matches, values, functions) {
   names(res) <- lapply(res, function(x) key(x$srcref))
 
   class(res) <- "line_coverages"
+
+  res
+}
+
+br_coverages <- function(source_file, matches, values, functions) {
+
+  src_file <- srcfilecopy(source_file, readLines(source_file))
+
+  line_lengths <- vapply(src_file$lines[as.numeric(matches$line)], nchar, numeric(1))
+
+  res <- Map(function (br, line, length, value, func) {
```

Comment:
I think you should stick with `lapply` so it is consistent.

Reply:
`Map` is from the original covr.